src/Expectation.php

```diff
@@ -159,7 +159,7 @@ public function toContain($needle): Expectation
     /**
      * Asserts that all of the provided $needles are elements of the value.
      *
-     * @param array $needles
+     * @param array{mixed} $needles
```

Comment:
This should be `array<mixed>`

`array{mixed}` describes the array shape. So it would mean in this array key `0` has `mixed` type value.